agent/engine/engine_unix_integ_test.go

```diff
@@ -778,19 +779,19 @@ func TestDockerStopTimeout(t *testing.T) {
 		t.Errorf("Expect the docker stop timeout read from environment variable when ECS_CONTAINER_STOP_TIMEOUT is set, %v", dockerTaskEngine.cfg.DockerStopTimeout)
 	}
 	testTask := createTestTask("TestDockerStopTimeout")
-	testTask.Containers = append(testTask.Containers, createTestContainer())
-	testTask.Containers[0].Command = []string{"sh", "-c", "while true; do echo `date +%T`; sleep 1s; done;"}
-	testTask.Containers[0].Image = testBusyboxImage
+	testTask.Containers[0].Command = []string{"sh", "-c", "trap 'echo hello' SIGTERM; while true; do echo `date +%T`; sleep 1s; done;"}
+	testTask.Containers[0].Image = "busybox"
```

Comment:
Why is the Image changed from `testBusyboxImage` to `"busybox"`?

Reply:
Forget to change it back, will update.